lib/train/platforms/detect/specifications/os.rb

```diff
@@ -21,7 +21,7 @@ def self.load
             is_windows = false
             is_windows = true if winrm?
 
-            if @backend.class.to_s == 'Train::Transports::Local::Connection'
+            if backend_name == 'Train::Transports::Local::Connection'
```

Comment:
Did you figure out where this methed is coming in from? I only see the above being included via Scanner and I get lost after that.